packages/sdk/identity/src/odis/bls-blinding-client.ts

```diff
@@ -36,8 +36,16 @@ export class WasmBlsBlindingClient implements BlsBlindingClient {
     }
   }
 
-  async blindMessage(base64PhoneNumber: string): Promise<string> {
-    const userSeed = randomBytes(32)
+  async blindMessage(base64PhoneNumber: string, seed?: Uint8Array): Promise<string> {
+    let userSeed
+    if (!seed) {
+      userSeed = randomBytes(32)
+      console.warn(
+        'Warning: Use a private deterministic seed (i.e. DEK private key) to preserve user quota when requests are replayed.'
+      )
```

Comment:
I don't think we should pushing warnings to the user console from this library code, especially because there are valid reasons not to use a fixed seed. I would suggest we note this in the doc-string on this function.